pandas/core/frame.py

```diff
@@ -4135,10 +4136,10 @@ def _set_value(
             else:
                 icol = self.columns.get_loc(col)
                 iindex = self.index.get_loc(index)
-            self._mgr.column_setitem(icol, iindex, value)
+            self._mgr.column_setitem(icol, iindex, value, inplace=True)
             self._clear_item_cache()
 
-        except (KeyError, TypeError, ValueError):
+        except (KeyError, TypeError, ValueError, LossySetitemError):
```

Comment:
can you add LossySetitemError to the comment on L4145

Reply:
Whoops, sorry for some reason I thought I already addressed this one